noir-projects/aztec-nr/aztec/src/macros/notes/mod.nr

```diff
@@ -405,10 +413,10 @@ comptime fn generate_multi_scalar_mul(
 ///             log_plaintext[32 + i] = note_type_id_bytes[i];
 ///         }
 ///
-///         let packed_note_content = [npk_m_hash as Field, randomness as Field];
+///         let packed_note = [npk_m_hash as Field, randomness as Field];
```

Comment:
I deprecated the use of the term "note content" as it no longer makes much sense.